### DISCUSSION TITLE
UIKIt and Storyboard

### DISCUSSION THREAD

**Discussion by rockcoder1920** [skiptools/skip#177] [Q&A] [answered]
Does skip supports UIKit and Storyboard or it only supports full SwiftUI apps? Can I target iOS 14 and make the app available for Android and iOS?

**Comment by marcprux** [comment, marked as answer]
Skip only provides a SwiftUI implementation for Android. While there is a little bit of UIKit support here and there in order to fill in the gaps (e.g., `UIImage.init(data:)`), there is no support for storyboards.

Additionally, Skip requires iOS 16 and higher.
